### PR TITLE
ALR-859 (Decision Matrix and Part B Assessment Questions)

### PR DESCRIPTION
**Pull request by siddharthKav00**

### CODE REVIEW

File: src/main/default/decisionMatrixDefinition/Assigned_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            <columnType>Output</columnType>
+            <dataType>Number</dataType>
+            <displaySequence>2</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>AssignedScore</name>
+        </columns>
+        <decisionMatrixDefinition>Assigned_Score_Decision_Matrix</decisionMatrixDefinition>
+        <label>Assigned Score Decision Matrix V1</label>
+        <startDate>2024-03-06T11:26:06.000Z</startDate>
+        <status>Draft</status>
```
Comment (andreyshestakov-phsa) [inline src/main/default/decisionMatrixDefinition/Assigned_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml:26]: Please clarify if this Decision Matrix must be deployed and kept as **inactive**, and if yes - then why?

File: src/main/default/decisionMatrixDefinition/Risk_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <columnType>Output</columnType>
+            <dataType>Number</dataType>
+            <displaySequence>3</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>RiskScore</name>
+        </columns>
+        <decisionMatrixDefinition>Risk_Score_Decision_Matrix</decisionMatrixDefinition>
+        <label>Risk Score Decision Matrix V1</label>
+        <startDate>2024-03-06T11:07:13.000Z</startDate>
+        <status>Draft</status>
```
Comment (andreyshestakov-phsa) [inline src/main/default/decisionMatrixDefinition/Risk_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml:32]: Please clarify if this Decision Matrix must be deployed and kept as **inactive**, and if yes - then why?

**Comment by andreyshestakov-phsa** [comment]
Pre-deployment step:
- Decision Matrices have been deployed into CI org
- Active Decision Matrix Versions inactivated, their csv files uploaded, and the Version activated back
- Inactive Decision Matrix Versions their csv files uploaded

Decision Matrices have been moved out of regular deployments into a new `src-bre` folder

File: src/main/default/decisionMatrixDefinition/Risk_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml (hunk 2 above)
Comment (siddharthKav00) [reply src/main/default/decisionMatrixDefinition/Risk_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml]: Status now changed to Active.

**Review by Sumancdey — APPROVED** [review]

**Pull request merged**